src/routes/users.tsx

```diff
@@ -1,12 +1,31 @@
-import { UserItem } from "../components/User/User.tsx";
+import { UserItem } from "../components/User/UserItem.tsx";
+import { config } from "../components/User/user.config.ts";
 import User from "../components/User/User.types.ts";
+import { useParams } from "react-router-dom";
+import { useGet } from "../hooks/useGet.ts";
 
-interface UsersList {
-  usersList: User[];
-}
+export function Users() {
+  const { userId } = useParams();
+  const { isLoading, isError, data } = useGet(config.url, userId);
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (isError) {
+    return <div>Error: Oh no!</div>;
+  }
+  //Changed this to be more meaningful name as "data" is not very descriptive
+  let usersData = data;
```

Comment:
No reason to do it really, but if you do - a better way is to re-assign on destructuring:

```
const { isLoading, isError, data: usersData } = useGet(...);
```